src/structuralFunctionality/language-switcher/support/utils.ts

```diff
@@ -334,9 +334,11 @@ export async function getT9nData(
 
 export function convertT9nToConfigData(
   data: { [key: string]: string },
-  base: ApplicationBase
+  base: ApplicationBase,
+  useStructuredClone?: boolean
 ): { [key: string]: any } {
-  const config = structuredClone(base.config);
+  const config =
+    useStructuredClone === false ? base.config : structuredClone(base.config);
```

Comment:
I think this makes more sense readability wise being the opposite way `useStructuredClone ? structuredClone(base.config) : base.config;`

I don't mind updating my apps